test/data/test_multi_embedding_tensor.py

```diff
@@ -99,6 +99,21 @@ def test_from_tensor_list():
     assert torch.allclose(met.offset, expected_offset)
     assert_equal(tensor_list, met)
 
+    met_with_nan = met.clone()
+    # Set some values to be na
+    met_with_nan.values[0::2] = -1
+
+    # Test fill_col
+    met_1 = met_with_nan.clone()
+    for col in range(met.num_cols):
+        met_1.fillna_col(col, 100)
+    assert not torch.all(met_1.values == -1).any()
+    expected_values = torch.tensor([
+        [100, 100, 100, 100, 100, 100],
+        [3, 4, 5, 8, 9, 11],
+    ])
```

Comment:
I am having some hard time understanding test, especially the `expected_values`.  Can we add a test not by comparing the `values`, but compare based on elements, like `met_1[i, col]`?

Also, can we test the column-wise `fillna_col`, instead of filling all columns with 100?

Reply:
sure